invoicer_project/static/js/registration_request.js

```diff
@@ -0,0 +1,269 @@
+        isPasswordValid = `Max length – ${passwordMaxLength} chars`;
+    } else if (!(/^[a-z0-9]+$/i.test(passwordToValidate))) {
+        isPasswordValid = "Only A-Z, a-z and 0-9";
+    } else if (!(/\d/.test(passwordToValidate))) {
+        isPasswordValid = "At least one number";
+    } else if (!(/[a-z]/.test(passwordToValidate))) { 
+        isPasswordValid = "At least one lowercase";
+    } else if (!(/[A-Z]/.test(passwordToValidate))) {
+        isPasswordValid = "At least one capital";
+    } else {
+        isPasswordValid = '';
+    }
+    return isPasswordValid;
+}
+
+function validateRegistration() {
+    removeAllErrorAttributes();
+    setMaxFieldContainerHeights();
+    let allFieldsValidationResult = {
+        'nameValidationResult': validateNameAndSurnameAsStrings(nameField.value),
+        'surnameValidationResult': validateNameAndSurnameAsStrings(surnameField.value),
+        'emailValidationResult': validateEmail(emailField.value),
+        'passwordValidationResult': validatePasswordAsString(passwordField.value),
+        'repeatPasswordValidationResult': validatePasswordAsString(repeatPasswordField.value),
+        'doPasswordsMatch': (() => {
+            if (passwordField.value !== repeatPasswordField.value) {
+                return "Passwords don't match!";
+            } else {
+                return '';
+            }
+        })()
+    };
+    return allFieldsValidationResult;
+}
+
+function setErrorAttributesToFields(errorsObject) {
+    if (errorsObject['nameValidationResult']) {
+        nameField.setAttribute("error", "true");
+        nameField.setAttribute("errorText", errorsObject['nameValidationResult']);
+    }
+    if (errorsObject['surnameValidationResult']) {
+        surnameField.setAttribute("error", "true");
+        surnameField.setAttribute("errorText", errorsObject['surnameValidationResult']);
+    }
+    if (errorsObject['emailValidationResult']) {
+        emailField.setAttribute("error", "true");
+        emailField.setAttribute("errorText", errorsObject['emailValidationResult']);
+    }
+    if (errorsObject['passwordValidationResult']) {
+        passwordField.setAttribute("error", "true");
+        passwordField.setAttribute("errorText", errorsObject['passwordValidationResult']);
+    }
+    if (errorsObject['repeatPasswordValidationResult']) {
+        repeatPasswordField.setAttribute("error", "true");
+        repeatPasswordField.setAttribute("errorText", errorsObject['repeatPasswordValidationResult']);
+    }
+    if (errorsObject['doPasswordsMatch']) {
+        passwordField.removeAttribute("errorText");
+        repeatPasswordField.removeAttribute("errorText");
+        passwordField.setAttribute("error", "true");
+        repeatPasswordField.setAttribute("error", "true");
+        passwordField.setAttribute("errorText", errorsObject['doPasswordsMatch']);
+    }
+}
+
+nameField.addEventListener('input', () => {
+    nameField.removeAttribute("errorText");
```

Comment:
You could create method like this
```js
const removeError(...fields) {
   for(let field of fields) {
       field.removeAttribute('error');
       field.removeAttribute('errorText');
   }
```

and reuse it: `removeErrors(nameField, passwordField)`

Reply:
thx for nice idea, fixed